### PR TITLE
fix: :bug: strongly typed Id parameter never allowing view with id param

### PR DESCRIPTION
**Pull request by Eimen2018**

### CODE REVIEW

File: packages/stacked_generator/test/router/route_class_generator_test.dart [hunk 1]
```diff
@@ -34,7 +34,7 @@ void main() {
     /// Note: It usually adds an extra line in the bottom of the result
     void generateRouteAndExpectResult(
         List<RouteConfig> routes, dynamic expectedResult,
-        {bool verbose = false}) {
```
Comment (eb-so) [inline packages/stacked_generator/test/router/route_class_generator_test.dart]: Why did you add this and not use it?

Comment (Eimen2018) [reply packages/stacked_generator/test/router/route_class_generator_test.dart]: It's leftover from the stash i use for generating the right constant 😁. Will remove it now

**Comment by eb-so** [comment]
LGTM
